presets/consumer/avformat/stills/JPEG

```diff
@@ -6,5 +7,7 @@ an=1
 audio_off=1
 g=1
 bf=0
+pix_fmt=yuvj422p
```

Comment:
I did not test this. But I wonder why this is not RGB8 - which would be the native pixel format for JPEG. Does this introduce an extra conversion to eventually get to RGB?

Reply:
`ffmpeg -h encoder=mjpeg` only supports YUV pixel formats.  I figured I would incur a conversion penalty if I handed RGB8 to the encoder?

Reply:
Why do you think JPEG is a RGB pixel format?

Reply:
I made an incorrect assumption. I learned something new today!

Reply:
My understanding was that old JPEGs were BT.601 and new JPEGs were sYCC from the sRGB spec.  Some "[light reading](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-T.871-201105-I!!PDF-E)" seemed to say that's generally true, except when it isn't, as is always the case when JPEG is involved. :)

Reply:
That’s about color space. My reading of things (not in ffmpeg) say encoding first converts RGB to Y’CbCr. See section 6.2 in the doc Austin linked.

Reply:
I think we're all saying the same thing at this point.  When I said BT.601 and sYCC, I was implying that the encoding was YCbCr as well, since those are YCbCr color spaces.